readthedocs-theme/templates/readthedocs/pricing.html

```diff
@@ -0,0 +1,504 @@
+                  </div>
+                </div>
+              </div>
+
+            </div>
+          </div>
+        </div>
+        <!-- end card -->
+        
+        <!-- Card -->
+        <div class="ui card">
+
+          <div class="content">
+            
+            <div class="ui label ribbon yellow big">Gold</div>
+
+            <div class="ui header huge"><small><i>from</i></small> $5</div>
+            <div class="meta">
+              <span class="category">per month</span>
+            </div>
+
+            <button class="ui button fluid large yellow">Subscribe</button>
+
+            <div class="description">
+              
+              <div class="ui list relaxed divided">
+                <div class="item">
+                  <div class="content">
+                    <div class="header">Automatic build</div>
+                    <div class="description">on every commit</div>
+                  </div>
+                </div>
+                <div class="item">
+                  <div class="content">
+                    <div class="header">Multi-version handling</div>
+                    <div class="description">from branches or tags</div>
+                  </div>
+                </div>
+                <div class="item">
+                  <div class="content">
+                    <div class="header">Downloadable formats</div>
+                    <div class="description">PDF, HTML</div>
+                  </div>
+                </div>
+                <div class="item">
+                  <div class="content">
+                    <div class="header">Enhanced full-text search</div>
+                  </div>
+                </div>
+                <div class="item">
+                  <div class="content">
+                    <div class="header">Ad-free user</div>
+                    <div class="description">stop seeing ads on any Read the Docs hosted documentation when logged in your account</div>
+                  </div>
+                </div>
+                <div class="item">
+                  <div class="content">
+                    <div class="header">Ad-free documentation</div>
+                    <div class="description">stop showing ads to all your users on your Read the Docs hosted documentation</div>
+                  </div>
+                </div>
+              </div>
+
+            </div>
+          </div>
+        </div>
+        <!-- end card -->
+        
+      </div>
+
+    </div>
+  </section>
+
+
+  <!-- Intro -->
+  <section class="ui vertical segment padded" id="intro">
+    <div class="ui container">
+
+      <div class="ui vertical segment basic padded">
+
+        <div class="ui grid center aligned">
+          <div class="column twelve wide computer sixteen wide mobile">
+
+            <h3 class="ui header massive"><span class="ui header">About our plans</span></h3>
+            <p>
+              Our plans are divided in two main types: the Community and the Enterprise plans.
```

Comment:
Enterprise isn't correct here, we refer to our commercial plans as "Read the Docs for Business". It might be best to replace this section with a link directly to the doc page we have on this already, or bring the copy from the doc page in here:

https://docs.readthedocs.io/en/stable/choosing-a-site.html#choosing-between-our-two-sites

Reply:
Do you mean replace the entire section, FAQ and all, with just a button/link to the docs?
I can also bring the same copy here but the format it is written in works best in straight line docs.

Reply:
I significantly reduced this section for now, leaving the title, changed to reflect the new _platforms_ concept, one short paragraph pulled from the docs page [choosing-between-our-two-sites](https://docs.readthedocs.io/en/stable/choosing-a-site.html#choosing-between-our-two-sites) and a link there.

Maybe we can comeback here when the copy is more mature and expand the marketing message.

Commit https://github.com/readthedocs/site-community/pull/34/commits/3a673c9660e20d06360fd29765eedeeea4cd67a0

Reply:
Also created issue #49 to keep track of this copy.

Reply:
Makes sense. We have a few issues to expand this page/content. #65 is another I just added